data/announcements/scheduled.yaml

```diff
@@ -0,0 +1,42 @@
+---
+# For an example of the format, see commented structure below.
+#
+# 🛈  Changes require approval from @kubernetes/steering-committee
+
+# The order matters: if two schedules overlaps, the announcement
+# that comes FIRST in the following list takes precedence.
+#
+#announcements:
+#  - startTime: 2020-01-01T00:00:00
+#    # startTime is optional
+#    endTime: 2021-04-01T00:00:00
+#    # endTime is required
+#    style: >-
+#      color: #fff; background: #000;
+#    # style is optional; if using, set both color and background
+#    # choose a dark color for the background
+#    title: "Sample 1 announcement"
+#    # title is optional
+#    message: |
+#      Message *one*.  
+#      [Hyperlink](https://en.wikipedia.org/wiki/Hyperlink).
+#    # message is required. You can use Markdown.
+#  - name: Sample 2
+#    startTime: 2020-01-01T00:00:00
+#    endTime: 2021-04-01T00:00:00
+#    message: |
+#      Message *two*.
+
+# leave the "announcements" key in place
+announcements:
+  - startTime: 2020-06-04T23:04:39
```

Comment:
@sftim,
Is there a good reason to keep the existing announcement or should this PR only include the example template?
I don't see a strong case to place an announcement with this PR.
An announcement can be added in a separate PR (and approved separately by the Steering Committee).